code/modules/events/disease_outbreak.dm

```diff
@@ -35,8 +56,31 @@
 			continue
 		disease_candidates += candidate
 
+/datum/event_admin_setup/disease_outbreak
+	///Admin selected disease, to be passed down to the round_event
+	var/virus_type
+
+/// Checks for candidates. Returns false if there isn't enough
+/datum/event_admin_setup/disease_outbreak/proc/candidate_check()
+	var/datum/round_event_control/disease_outbreak/disease_control = event_control
+	disease_control.generate_candidates() //can_spawn_event() is bypassed by admin_setup, so this makes sure that the candidates are still generated
+	return length(disease_control.disease_candidates)
+
+/datum/event_admin_setup/disease_outbreak/prompt_admins()
+	var/candidate_count = candidate_check()
+	if(!candidate_check())
```

Comment:
```suggestion
	var/candidate_count = candidate_check()
	if(!candidate_count)
```

Again, only need to call candidate_check() once. (although tit was a different proc the other time)